modules/sort/test/spec.coffee

```diff
@@ -108,3 +141,52 @@ describe 'hx-sort tests', ->
         if supportsOptions()
           array.sort(hx.sort.localeCompare('sv')).should.eql(['a', 'ä', 'e', 'é', 'è', 'z'])
           array.sort(hx.sort.localeCompare('de')).should.eql(['a', 'e', 'é', 'è', 'z', 'ä'])
+
+  describe 'using localeCompare potentially with Intl.Collator. ', localeCompareTests
+
+  describe 'using localeCompare without Intl.Collator. ', runWithoutCollator(localeCompareTests)
+
+  compareNullsLastTests = ->
+    it 'Strings should return the right value', ->
+      hx.sort.compareNullsLast('a', 'b').should.equal(-1)
+      hx.sort.compareNullsLast('b', 'a').should.equal(1)
+      hx.sort.compareNullsLast('a', 'a').should.equal(0)
+
+    it 'undefined should always be sorted to the end', ->
+      hx.sort.compareNullsLast(undefined, 'a').should.equal(1)
+      hx.sort.compareNullsLast('a', undefined).should.equal(-1)
+      hx.sort.compareNullsLast(undefined, 'z').should.equal(1)
```

Comment:
Is there a danger of api function proliferation here ? Would an options object help(eg `hx.sort.compare({ locale: true })(0, 1)` ? And then deprecate hx.sort.localeCompare